linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_password_pam_unix_remember/ansible/shared.yml

```diff
@@ -1,21 +1,29 @@
-# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle,multi_platform_ubuntu
 # reboot = false
 # strategy = configure
 # complexity = low
 # disruption = medium
 
-{{% if product in [ "sle12", "sle15" ] %}}
+{{% if product in [ "sle12", "sle15" ] or product in [ "ubuntu1804", "ubuntu2004" ] %}}
 {{%- set accounts_password_pam_unix_remember_file = '/etc/pam.d/common-password' -%}}
 {{% else %}}
 {{%- set accounts_password_pam_unix_remember_file = '/etc/pam.d/system-auth' -%}}
 {{% endif %}}
 
 {{{ ansible_instantiate_variables("var_password_pam_unix_remember") }}}
 
+{{% if product in ['ubuntu2004'] %}}
+{{{ ansible_ensure_pam_module_line(accounts_password_pam_unix_remember_file,
+                                          "password",
+                                          "[success=1 default=ignore]",
```

Comment:
I don't like this hardcoded config, it doesn't allow for custom configs. Different users have different configurations. Check the bash remediation